cmd/skopeo/delete.go

```diff
@@ -17,7 +17,9 @@ type deleteOptions struct {
 }
 
 func deleteCmd(global *globalOptions) cli.Command {
-	sharedFlags, sharedOpts := sharedImageFlags()
+	// delete has no shared flags or opts
+	var sharedFlags []cli.Flag
+	sharedOpts := &sharedImageOptions{}
```

Comment:
This is to avoid defining the `--authfile` flag twice, I expect?

If so, doing it this way is risky because adding any new flags to `sharedImageFlags` would not be reflected here (and it would not be possible to fix cleanly).

Rather, I think `sharedImageFlags` should remain as is, and `dockerImageOptions` should _optionally_ add the specific `--authfile` flag only if the prefix is non-empty.

Reply:
I thought about this as well but wanted to keep the logic for shared flags, simple. However, now code is adjusted according to your suggestions.